utility_network/trace-a-utility-network/src/main/java/module-info.java

```diff
@@ -0,0 +1,9 @@
+module trace.a.utility.network.main {
+  requires com.esri.arcgisruntime;
+  requires javafx.base;
+  requires javafx.controls;
+  requires javafx.fxml;
+  requires javafx.graphics;
```

Comment:
I would put a blank line between the `requires` and the `exports`. I think it makes it more readable separating the different kinds of entries.